jwst/extract_1d/extract.py

```diff
@@ -3736,6 +3736,13 @@ def create_extraction(extract_ref_dict,
         use_source_posn = False
         log.info(f"Setting use_source_posn to False for source type {source_type}")
 
+    # Turn off use_source_posn if working on non-primary NRS fixed slits
```

Comment:
I am not sure why these five lines are showing up.  I did not make these edits, and I freshly pulled the jwst master branch just before committing and submitting the PR.

Reply:
It's probably a merging issue.  Those lines were removed in #8467, which touched many parts of the pipeline code.